src/util.h

```diff
@@ -72,7 +75,17 @@ uint64_t GetTimeStampUS(void);
 // refer to https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique
 template <typename T, typename... Args>
 std::unique_ptr<T> MakeUnique(Args&& ... args) {
-     return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+  return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
 }
 
+struct UniqueIterator : std::unique_ptr<rocksdb::Iterator> {
```

Comment:
Should we reveal on the name that this is the Rocksdb iterator?

Reply:
Now it is `DBUtil::Iterator`, more clear.